src/lua-format.cpp

```diff
@@ -18,7 +54,9 @@ string __format(ANTLRInputStream& input, const Config& config) {
     }
 
     vector<antlr4::Token*> tokenVector;
-    for (auto t : tokens.getTokens()) {
+    for (auto t : tokenstream.getTokens()) {
```

Comment:
This and `parseStringToken ` should be moved into `FormatVisitor.cpp`.
Override `visitString` in LuaBaseVisitor to transform string ast node.

Reply:
yes, makes more sense